widget/playground/src/containers/DefaultChainAndToken/DefaultChainAndToken.tsx

```diff
@@ -82,6 +82,7 @@ export function DefaultChainAndToken({ type }: { type: Type }) {
         title="Default Blockchain"
         hasLogo={true}
         placeholder="Chain"
+        disabled={!blockchains?.length}
```

Comment:
I added the `disabled` property to the `FunctionalLayout.Liquidities.tsx` but it is not needed to add it to `FunctionalLayout.Wallets.tsx` because the `MultiSelect` component in that file does not have any dependency on the meta.